### ISSUE TITLE
A model connecting host galaxy properties with multiple supernova populations

### ISSUE THREAD

**Issue by kponder** [kbarbary/snpgm#13]
Here's an excerpt from Phil and my notes:

Several studies in recent years have pointed out correlations between type Ia SNe Hubble diagram residuals and the properties of their host galaxies, including stellar mass and morphological type at cosmological redshifts, and local star formation rates at low redshift. 

At redshifts 0.1 - 1.0, the available information on the host galaxy is limited, but we can expect to be given posterior samples in M*, SFR, and other SED parameters, and also in structural parameters such as size, ellipticity and bulge/disk ratio. Taking these cartoon models seriously, we may be able to make rough approximations to local host properties, by using the model to interpolate the colors and brightnesses to the SN position. 

There is some evidence for multiple sub-populations of supernova, whose intrinsic properties we might expect to depend differently on host galaxy parameters. The relative abundances of the populations will evolve over cosmic time, if they are coupled to the host properties.

So, here's what we're planning to do: 
- [ ] 1. Write down a generic PGM for the above
- [ ] 2. Focus it on a possible simple implementation, anticipating having access to posterior samples in both SN and host parameters.
- [ ] 3. Write out the probability theory corresponding to this model, and determine good sampling options to pursue. 
- [ ] 4. Design a small simulated dataset and analyze it using this model. 

Comments welcome!

**Comment by drphilmarshall** [comment]
Hey @wmwv, perhaps the Bildsten and Scannapieco A/B rates idea might have
some legs in guiding this particular model, what do you think?

**Comment by rbiswas4** [comment]
Hi @kponder,

>  we can expect to be given posterior samples in M*, SFR, and other SED parameters, and also in structural parameters such as size, ellipticity and bulge/disk ratio. 

I am not sure I am understanding you correctly but I interpret your statements in the following way:

P(SN probability / volume /time  \vert z ) = P(SN probability  / time / volume | Host galaxy parameters) P( Host galaxy parameters |z ) number density of galaxies (z) \* Volume (z)

Of course you will need the extra piece P(SN parameters \vert Host Galaxy parameters)  with this.

It would be interesting to talk about this, as I think I am trying to do part of the simulation step which may be of help to you? The way I am getting P( Host galaxy parameters \vert z) is through  the LSST catsim database. Of course, this is not real data, and will need improvements, but I am hoping that this is a product that will improve and later come to represent our best understanding. That is my logic for using this framework.

**Comment by wmwv** [comment]
Yes, but ideally with local measures of star formation (Rigault+13).  Restframe NUV is substantially more informative than restframe u (Rigault+15, Kelly+15).

The obviously tempting story is that there are different populations from short delay time and long delay time SNeIa.  This maps strongly to local projected measures of star formation, and more loosely to global star formation measures.

The Bildsten+Scannapieco model is consistent with such a story.

It doesn't, however, provide a physically-motivated prediction from first principles.  Rather, it provides a transfer function from a given star-formation history (stellar age distribution) to an expected rate.

This transfer function does, however, encode the information about the relative efficiencies of each channel.  Together with the stellar age distribution, this should then result in a prediction for n_A(z) and n_B(z).
- Michael

**Comment by kbarbary** [comment]
Kara's plan sounds good to me.

Regarding rate models, as @wmwv said, the A+B model captures the rough behavior (younger stars = higher SN rate) but isn't really physical. A better model is `sn_rate(t) ~ t^-1` for t > 40 Myr stellar age. This is physically motivated (for the double degenerate model) and current measurements are consistent with it. In rates at least, there's not much evidence for "two populations" rather than a continuum.

**Comment by drphilmarshall** [comment]
That's interesting. Sounds like continuing to work on model comparison
statistics should be part of our plan.

With regard to the local vs global measurements: what's a good dataset in
which to investigate "semi-local" measurements, like the Sersic models we
expect from LSST? Has anyone been thinking about experiments like taking
nearby galaxies with local measurements and "observing" them at higher
redshift with PhoSim or some other simulation tool? Or do we already know
how broadband surface brightness at the SN position relates to local SFR?
